graphql/schema.graphql

```diff
@@ -498,6 +504,31 @@ input TaskSyncOptionsInput {
   patchEnabled: Boolean!
 }
 
+input BuildBaronSettingsInput {
+  ticketCreateProject: String!
+  ticketSearchProjects: [String!]
```

Comment:
This line indicates this field can be null or an array with String elements. But the rest of the fields are all fully required